Makefile

```
@@ -238,6 +239,7 @@ clean-tools:
 mostlyclean: tidynonmodern tidymodern
 	rm -f $(SAMPLE_SUBDIR)/*.bin
 	rm -f $(CRY_SUBDIR)/*.bin
+	rm -f $(PHONE_SUBDIR)/*.bin
```

Having 3 rules for effectively the same thing is silly, and breaks again if a new subfolder is added.
```suggestion
	find sound -iname '*.bin' -exec rm {} +
```